pdf-generator/POSTINSTALL.md

```diff
@@ -16,24 +17,26 @@ Update your Firebase Storage security rules to allow reads from template path:
 If `OUTPUT_STORAGE_BUCKET` is set, you need to allow writes to where generated pdf files are going to be stored too:
 
     match /b/${param:OUTPUT_STORAGE_BUCKET}/o {
-      // Allow access to the path where generated pdf files will be stored
+      // Allow access to the generated pdf file path
       match /{outputs_path} {
         allow write;
       }
     }
 
 Check [here](https://firebase.google.com/docs/storage/security) for more information about security rules.
 
-***
+---
```

Comment:
My markdown linter [remark](https://github.com/remarkjs/remark) keeps changing `---` to `***`, is it important to use `---`? If yes I should be able to find a way to configure remark.

Reply:
Is this still an issue, hopefully this configurable as it seems fairly consistent to have *** in most of Extensions I have worked on so far?

Reply:
Seems like remark is very customizable, it has a configuration mechanism like eslint rules: https://github.com/remarkjs/remark-lint#rules
So all should be fine.